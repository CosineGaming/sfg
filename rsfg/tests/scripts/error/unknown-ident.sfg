fn main()
	var x = y
	var rec = 2

