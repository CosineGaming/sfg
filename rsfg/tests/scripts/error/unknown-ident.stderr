[ERROR] referenced unknown identifier y at 2:10